skills/csharp/tests/emailskill.tests/Flow/DeleteEmailFlowTests.cs

```diff
@@ -31,7 +31,6 @@ await this.GetTestFlow()
                 .Send(BaseTestUtterances.FirstOne)
                 .AssertReply(this.DeleteConfirm())
                 .Send(GeneralTestUtterances.No)
-                .AssertReplyOneOf(this.CancelResponses())
                 .StartTestAsync();
         }
 
```

Comment:
Could we add some new tests? Since it is 'new delete email flow'..

Reply:
updated test